### ISSUE TITLE
Add Java section?

### ISSUE THREAD

**Issue by vinhhv** [anthonyprograms/ace-it#9]
Should we add a Java section? Since C++ and Java are the main choices for solving these type of questions. Eventually we can add a Python section as well.

**Comment by anthonyprograms** [comment]
Yeah, I'm going to wait until we meet up to do anymore so i can just understand how everything is working so far. Then I will continue
